### PR TITLE
Release Manager Actions Checklist Template

### PR DESCRIPTION
**Pull request by tsande16**
This is a template to assist the release manger in the actions that are to be performed for each release. At the moment, the GH actions to copy the template for the next release do not exist. I looked into it and we may want to implement that capability for the next release. For now it is done manually.

### CODE REVIEW

File: docs/release/release-actions-0.5.0.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Release Manager Actions Checklist 0.5.0
+
+## Release Process Overview
+This is the full detailed release process, including the steps that are performed by the GitHub automation: [Release](https://github.com/eclipse-pass/main/blob/main/docs/dev/release.md)
+
+## Pre-release
+
+- [ ] Ensure all code commits and PRs intended for the release have been merged.
```
Comment (bbranan) [inline docs/release/release-actions-0.5.0.md:8]:
```suggestion
- [ ] Identify the version to be utilized for the release.
- [ ] Ensure all code commits and PRs intended for the release have been merged.
```

**Review by jabrah — APPROVED** [review]

File: docs/release/release-actions-template.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- [ ] Pass Acceptance Testing Release Page - Perform after the Pass Acceptance Testing release is complete - [Pass Acceptance Testing GitHub Release Page](https://github.com/eclipse-pass/pass-acceptance-testing/releases)
+
+## Release Other Projects
+
+- [ ] Release Pass Docker - [Release workflow](https://github.com/eclipse-pass/pass-docker/actions/workflows/release.yml)
+
+## Post-release
+
+- [ ] Test the release by using the newly updated pass-docker to run the release locally.
+- [ ] Draft release message and have technical & community lead provide feedback.
```
Comment (bbranan) [inline docs/release/release-actions-template.md:39]:
```suggestion
- [ ] Write release notes in the [Release Notes doc](../docs/release-notes.md), submit a PR for the changes, and ensure the PR is merged. Release Notes should be written so as to be understandable by community members who are not technical.
- [ ] Draft release message and have technical & community lead provide feedback. Ensure that a link to the release notes is included in the release message.
```

**Review by bbranan — CHANGES_REQUESTED** [review]
The overall structure and layout of this looks good. I'd suggest adding a step in docs/dev/release.md to identify the need to create a release page from the template created in this PR.

 As you use this for the 0.5.0 release make notes of any steps you take that aren't here and add them to the template for next time.

File: docs/release/release-actions-0.5.0.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Release Manager Actions Checklist 0.5.0
+
+## Release Process Overview
+This is the full detailed release process, including the steps that are performed by the GitHub automation: [Release](https://github.com/eclipse-pass/main/blob/main/docs/dev/release.md)
```
[the remark this answers is not in the document]
Comment (tsande16) [reply docs/release/release-actions-0.5.0.md:4]: Can they be absolute paths relative to the main the repo? For example I remember this from the review Andrew did for the GH automation: https://github.com/eclipse-pass/main/pull/522#discussion_r1170368684

So as long as I did [Release](/docs/dev/release.md), that should be fine?

[the remark this answers is not in the document]
Comment (tsande16) [reply docs/release/release-actions-0.5.0.md:4]: No worries. I will proceed with relative so that github and github pages work properly.

**Comment by tsande16** [comment]
Made the urls relative and performed a rebase to squash commits but still captures @bbranan edits.

**Comment by bbranan** [comment]
@tsande16 changes all look good. One last thing: I'd suggest adding a step in docs/dev/release.md to identify the need to create a release page from the template created in this PR. That way we remember to create and use this checklist every time.

**Review by bbranan — APPROVED** [review]

**Pull request merged**